internal/pkg/featureflags/disable.go

```diff
@@ -0,0 +1,52 @@
+package featureflags
+
+import (
+	"strings"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/exp/slices"
+
+	v1 "github.com/confluentinc/cli/internal/pkg/config/v1"
+	dynamicconfig "github.com/confluentinc/cli/internal/pkg/dynamic-config"
+	ppanic "github.com/confluentinc/cli/internal/pkg/panic-recovery"
+)
+
+func DisableHelpText(command *cobra.Command, flags []string) {
+	if len(flags) == 0 {
+		command.Hidden = true
+	} else {
+		formattedFlags := ppanic.ParseFlags(command, flags)
+		for _, flag := range formattedFlags {
+			_ = command.Flags().MarkHidden(flag)
```

Comment:
what's being ignored here?

Reply:
Basically, if we accidentally misspell a flag or add a flag that doesn't exist in launchDarkly, ignoring the error here from `MarkHidden` means the other flags will still be marked disabled like normal.